frontend/src/App.js

```diff
@@ -1,33 +1,28 @@
-/* eslint-disable class-methods-use-this */
-import React, { Component } from 'react';
+import React from 'react';
 import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
 import './App.css';
-import Home from './Home.jsx';
-import Experiment from './Experiment.jsx';
-import Status from './Status.jsx';
+import Status from './Status';
+import Home from './Home';
+import Experiment from './Experiment';
 
-class App extends Component {
-  render() {
-    return (
-      <div className='App'>
-        <Router>
+const App = () => {
+  return (
+    <div className="App">
+      <Router>
+        <div>
           <div>
-            <div>
-              <Link to="/">Home</Link>
-              <span> </span>
-              <Link to="/experiment">Prototype</Link>
-              <span> </span>
-              <Link to="/status">Status</Link>
-            </div>
-            <hr />
-            <Route exact path="/" component={Home} />
-            <Route path="/experiment" component={Experiment} />
-            <Route path="/status" component={Status} />
+            <Link to="/">Home</Link>
+            <Link to="/experiment">Prototype</Link>
+            <Link to="/status">Status</Link>
           </div>
```

Comment:
Is that normal that the page names don't have any space between each others?

![image](https://user-images.githubusercontent.com/23021242/51979451-2e5d4880-245b-11e9-8329-ea864b64e894.png)

Reply:
It's not great but we're going to be deleting this navigation soon anyways so I'm not too worried about it.